src/grok.ts

```diff
@@ -0,0 +1,59 @@
+import OpenAI from "openai";
+import { ChatCompletionCreateParamsNonStreaming } from "openai/resources/chat/completions";
+import { flow } from "./flow";
+const openai = new OpenAI({
+  apiKey:
+    "xai-dC1UY39OHowpobkG1XsK5dRdul9arYZFCFtfjTGLnZGOkXSYPQZIf83i0gQRz0aR3nGrpTiBsT0ywwyH",
```

Comment:
I don't think you intend to check this in.

It looks like this entire file is for testing and shouldn't be checked in.